src/main.ts

```diff
@@ -3,8 +3,9 @@ import * as exec from '@actions/exec'
 
 async function run() {
     try {
+        const inline: string = core.getInput('inline')
         // Run Pkg.test
-        await exec.exec('julia', ['--color=yes', '--check-bounds=yes', '--project', '-e', 'using Pkg; Pkg.test(coverage=true)'])
+        await exec.exec('julia', ['--color=yes', '--check-bounds=yes', '--inline=${inline}', '--project', '-e', 'using Pkg; Pkg.test(coverage=true)'])
```

Comment:
I think you have to use backticks here for interpolation

Reply:
Done